tests/phpunit/unit-tests/user/class-llms-test-user-permissions.php

```diff
@@ -164,6 +164,97 @@ public function test_user_can_manage_user_multiple_roles() {
 
 	}
 
+	/**
+	 * Test the editable_roles() filter for users with single roles.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_editable_roles_single_role() {
+
+		$users = $this->create_mock_users();
+
+		$all_roles = wp_roles()->roles;
+
+		$editable_roles = LLMS_Unit_Test_Util::call_method( $this->obj, 'get_editable_roles');
+
+		wp_set_current_user( $users['lms_manager'] );
+		$lms_manager_editable_roles = array_keys ( LLMS_Unit_Test_Util::call_method( $this->obj, 'editable_roles', array( $all_roles ) ) );
+
+		// Assert that lms_managers can edit mapped roles
```

Comment:
```suggestion
		// Assert that lms_managers can edit mapped roles.
```
I think my previous review included both the first word capitalization and the ending dot...
am I wrong?

Reply:
Oh sorry, didn't notice, will fix it